recipes/pycurl/pycurl.inc

```diff
@@ -0,0 +1,15 @@
+DESCRIPTION = "PycURL is a Python interface to libcurl."
+LICENSE = "LGPL-2.1"
+
+inherit python-module
+
+DEPENDS = "libcurl curl-dev native:util/curl-config libssl"
```

Comment:
Why curl-dev?  Best-practise is to only depend on the library name (libcurl in this case).  The library package should then take care to pull in additional dependencies.  If libcurl is broken in this way, it should be fixed.  If (when) curl-dev is not actually needed (because librcurl pulls in any needed dependencies), you must remove it here.